src/components/MajorFilter.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { Button, Autocomplete, Popover, Chip, TextField } from '@mui/material';
+import { MAJORS } from '../types/options';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+
+function MajorFilter() {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [majors, setMajors] = useState([]);
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <>
+      <Button
+        variant="outlined"
+        onClick={(e) => setAnchorEl(e.currentTarget)}
+        endIcon={<ArrowDropDownIcon color="primary" />}
+        sx={{ margin: 1, height: (theme) => theme.spacing(4) }}>
+        Majors
+      </Button>
+      <Popover
+        open={Boolean(anchorEl)}
+        anchorEl={anchorEl}
+        onClose={() => setAnchorEl(null)}
+        anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
+        transformOrigin={{ vertical: 'top', horizontal: 'left' }}
+        sx={{ maxWidth: 400 }}>
+        {majors.map((major) => (
+          <Chip
+            label={major}
+            variant="outlined"
+            color="primary"
+            key={major}
+            sx={{ mx: 1, mt: 1, color: '#000' }}
+          />
+        ))}
+        <Autocomplete
+          multiple
+          open={isOpen}
+          value={majors}
+          onChange={(e, val) => setMajors(val)}
+          onInputChange={(e, val) => {
+            if (val.length > 0) setIsOpen(true);
+            else setIsOpen(false);
```

Comment:
prevents listbox from showing when there is nothing in the input field like below,

<img width="556" alt="Screen Shot 2022-01-04 at 2 44 29 PM" src="https://user-images.githubusercontent.com/40483569/148136136-fba47758-378a-477d-af88-79be248b75ff.png">

Reply:
Maybe instead we can initialize this to false and show that dropdown arrow that toggles showing/hiding the list? The ones in the [playground](https://mui.com/components/autocomplete/#playground) have a button on the right to hide.

Reply:
Added the dropdown arrow, the only "problem" is that the arrow flips up even when the input has no options.

<img width="391" alt="Screen Shot 2022-01-06 at 9 10 18 PM" src="https://user-images.githubusercontent.com/40483569/148495082-6928b5b5-0a0a-4e7a-b5d6-dfd50954349c.png">

Reply:
Update:
removing`onInputChange` will avoid this bug but it will only show the options IF the drop down is clicked.